public/js/safetymaps/modules/incidents/VehicleIncidentsController.js

```diff
@@ -116,7 +116,9 @@ function VehicleIncidentsController(options, featureSelector) {
     if(me.options.incidentSource === "VrhAGS" || me.options.incidentSourceFallback === "VrhAGS") {
         // Initialize AGS service
         me.service = new AGSIncidentService(me.options.apiPath + "vrhAGS", me.options.apiPath + "vrhAGSEenheden");
-        me.initializeService();
+        me.initializeService(function() {
+            me.options.showStatus && me.updateStatus();
```

Comment:
Why is this necessary, `showStatusVrhAGS` sets a timeout in the `always()` for the `service.getVoertuigStatus()` promise so it is retried when it is initialized? If needed anyway use a Promise instead of a callback.

Reply:
@matthijsln see 7588ebf